### ISSUE TITLE
Maps still grey

### ISSUE THREAD

**Issue by Kinad** [mylen/leaflet.TileLayer.WMTS#7]
Hello,
I have a problem with your plugins, my map still grey, it doesn't load the wmts data, even if I copy/paste exactly your code of [http://leaflet.melard.fr](http://leaflet.melard.fr).
Can someone help me ?

**Comment by mylen** [comment]
You need to register an API key for your targeted domain. Did you do that?
You can also get a free developer key if you work locally though.

**Comment by Kinad** [comment]
Yes, I generate a free API key cause I just start learning Leaflet and I only work with local files. Maybe I don't understand what "local" mean... : if my html are on my computer, that would be ok ?

**Comment by mylen** [comment]
yes it should, you can always look in the browser inspector the network access in order to see what's wrong
